src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -3162,9 +3174,8 @@ subroutine PhotosynthesisHydraulicStress ( bounds, fn, filterp, &
                kp25_sha = kp25top * nscaler_sha
 
                ! Adjust for temperature
-
-               vcmaxse = 668.39_r8 - 1.07_r8 * min(max((t10(p)-tfrz),11._r8),35._r8)
-               jmaxse  = 659.70_r8 - 0.75_r8 * min(max((t10(p)-tfrz),11._r8),35._r8)
+               vcmaxse = 486.0_r8
```

Comment:
If these become hardcoded values, we should put them on the params file. And possibly the old hard coded constants should as well, and then a namelist trigger between the two.

Reply:
This is also part of a `use_Leuning` switch, if it's made